Jenkinsfile

```diff
@@ -53,7 +53,6 @@ pipeline {
                     env.lint_result = "FAILURE"
                 }
                 bbcGithubNotify(context: "lint/flake8", status: "PENDING")
-                // Run the linter, excluding build directories (this can also go in the .flake8 config file)
```

Comment:
Could also remove a comment at line 6 about python 2.7 and a comment at line 90 about using /tmp working directory.